install_pim/docker/installation_docker.rst

```diff
@@ -11,15 +11,47 @@ Akeneo maintains its own Docker images in https://github.com/akeneo/Dockerfiles.
 
    These instructions are valid for community edition as well as the enterprise edition.
 
-.. note::
 
-   These instructions assume you already have Docker and Docker Compose installed, in their most recent version. If not, please refer `to this documentation <https://github.com/akeneo/Dockerfiles/blob/master/Docs/getting-started.md>`_.
+System requirements
+-------------------
+
+Docker and Docker Compose
+*************************
+
+If you don't already have Docker and Docker Compose installed on your system, please refer to `the documentation of the GitHub repository <https://github.com/akeneo/Dockerfiles/blob/master/Docs/getting-started.md>`_.
+
+Setting up your host user
+*************************
+
+The PIM is shared with the containers as `a volume <https://docs.docker.com/engine/admin/volumes/volumes/>`_.
+The fpm and node containers will have write access to the PIM folder, and they will do so through their respective users: ``docker`` for fpm and ``node`` for node.
+
+These users UID and GID are both 1000:1000, so on Linux hosts **it is mandatory that the user of your host machine have 1000:1000 as UID and GID too**, or you'll end up with a non working PIM.
 
+You will not have this problem on Mac OS and Windows hosts, as those systems use a VM between the host and Docker, which already use a user with 1000:1000 as IDs.
```

Comment:
"not have" > "you won't face this problem" (just to avoid using always "to have")